CRM/Financial/BAO/FinancialType.php

```diff
@@ -217,27 +217,33 @@ public static function permissionedFinancialTypes(&$permissions, $descriptions)
       return FALSE;
     }
     $financialTypes = CRM_Contribute_PseudoConstant::financialType();
-    $prefix = ts('CiviCRM') . ': ';
-    $actions = ['add', 'view', 'edit', 'delete'];
+    $actions = [
+      'add' => ts('add'),
+      'view' => ts('view'),
+      'edit' => ts('edit'),
+      'delete' => ts('delete'),
+    ];
```

Comment:
I believe all these strings should be uppercased, as that's the reason for the test failure. So 

```suggestion:-6+6
     $actions = [
      'add' => ts('Add'),
      'view' => ts('View'),
      'edit' => ts('Edit'),
      'delete' => ts('Delete'),
    ];

Reply:
It would then break another part of the test, ex where it expects: "CiviCRM: edit contributions [...]", but then I could change that one for "CiviCRM: Edit contributions [..]" in the test?

Reply:
That sounds good.